--- FindBOOST.cmake	2016-06-18 10:57:20.079329627 +0200
+++ FindBOOSTNew.cmake	2016-06-18 10:57:41.907329500 +0200
@@ -11,7 +11,7 @@
 #############################################################
 IF( CMAKE_SIZEOF_VOID_P EQUAL 8 )
  message( STATUS "Will 'Find' Boost by setting it to the afs install of ilcsoft v01-17 64bit")
- SET( BOOST_DIR "/afs/desy.de/project/ilcsoft/sw/x86_64_gcc41_sl5/v01-17/LCFIVertex/v00-06-01/include" )
+ SET( BOOST_DIR "/usr/include" )
 ENDIF()
 
 
